webserver/js/index.js

```diff
@@ -11,16 +11,15 @@ let uid_text;
 document.addEventListener('DOMContentLoaded', function() {
     $('select').select2();
 
-    user_select = document.querySelector('#user_user');
+    user_select = $('#user_user');
     value_input = document.querySelector('#amount');
     qrcode = document.querySelector('#qrcode');
     qrcode_hint = document.querySelector('#qrcode_hint');
     uid_text = document.querySelector('#uid');
 
-    user_select.addEventListener('change', update_qr);
-    value_input.addEventListener('change', update_qr);
+    $('select').on('select2:select', update_qr);
+    value_input.addEventListener('onchange', update_qr);
```

Comment:
This looks wrong. The event is named `change` AFAIK, using `onchange` would look like this: `value_input.onchange = update_qr;` (but doing it that way isn't a good idea IMO)

Reply:
Correct, it should be should only be "change".